client/src/components/App.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useCallback } from "react";
+
+import enTranslations from "@shopify/polaris/locales/en.json";
+import { AppProvider, Page } from "@shopify/polaris";
+import { NewPageForm } from "./NewSurveyForm";
+import { NewSurveyModal } from "./NewSurveyModal";
+import { SurveyList } from "./SurveyList";
+
+export function App() {
```

Comment:
how does this connect to the `index.html`?

Reply:
In `client/src/index.js` we call the following which hooks into the `index.html`

```
ReactDOM.render(<App />, document.getElementById("root"));
```